src/Command/CheckAllSchemaTemplatesNameFieldsCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jobcloud\SchemaConsole\Command;
+
+use Jobcloud\SchemaConsole\Helper\SchemaFileHelper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CheckAllSchemaTemplatesNameFieldsCommand extends Command
+{
+    private const TYPES_FOR_VALIDATION = [
+        'record',
+        'enum',
+        'fixed'
+    ];
+
+    private const REGEX_MATCH_FIRST_LETTER = '/^[A-Za-z_]/';
+
+    private const REGEX_MATCH_STRING = '/^[A-Za-z0-9_]+$/';
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('kafka-schema-registry:check:template:name:field:all')
+            ->setDescription('Checks if name field follows avro naming convention')
+            ->setHelp('Checks if name field follows avro naming convention')
+            ->addArgument(
+                'schemaTemplateDirectory',
+                InputArgument::REQUIRED,
+                'Path to avro schema template directory'
+            );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        /** @var string $directory */
+        $directory = $input->getArgument('schemaTemplateDirectory');
+        $avroFiles = SchemaFileHelper::getAvroFiles($directory);
+
+        $io = new SymfonyStyle($input, $output);
+
+        $failed = [];
+
+        if (false === $this->checkSchemaTemplateNameFields($avroFiles, $failed)) {
+            $io->error('Following schema templates have invalid name field:');
+            $io->listing($failed);
+
+            return 1;
```

Comment:
i prefer to work with constants, e.g. const NO_ERROR = 0, ERROR = 1. But, like for the other comment, the other commands are the same, so unless we want to change all of them, let's leave it.